docs/source/user-guide/sql/write_options.md

```diff
@@ -53,26 +53,18 @@ Finally, options can be passed when running a `COPY` command.
 ```sql
 COPY source_table
 TO 'test/table_with_options'
-(format parquet,
-compression snappy,
-'compression::col1' 'zstd(5)',
-partition_by 'column3, column4'
+OPTIONS (
+  format parquet,
+  compression snappy,
+  'compression::col1' 'zstd(5)',
+  partition_by 'column3, column4'
 )
 ```
 
 In this example, we write the entirety of `source_table` out to a folder of parquet files. One parquet file will be written in parallel to the folder for each partition in the query. The next option `compression` set to `snappy` indicates that unless otherwise specified all columns should use the snappy compression codec. The option `compression::col1` sets an override, so that the column `col1` in the parquet file will use `ZSTD` compression codec with compression level `5`. In general, parquet options which support column specific settings can be specified with the syntax `OPTION::COLUMN.NESTED.PATH`.
 
 ## Available Options
 
-### COPY Specific Options
-
-The following special options are specific to the `COPY` command.
```

Comment:
These options are now specified directly in the DML syntax itself, so I removed them from here